src/views/Cart/Cart.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import SinglePosition from "./SinglePosition/SinglePosition";
+import woocommerceServices from "../../services/woocommerceService";
+import "./Cart.scss";
+
+const Cart = () => {
```

Comment:
Ten komponent to jakiś placeholder? 
Jak tak to chyba nie powinno go być w commicie